integration/test/installer/native.go

```diff
@@ -0,0 +1,40 @@
+package installer
+
+import (
+	"goharvest2/integration/test/utils"
+	"log"
+)
+
+type Native struct {
+	path string
+}
+
+func (r *Native) Init(path string) {
+	r.path = path
+}
+
+func (r *Native) Install() bool {
+	harvestFile := "harvest.yml"
+	harvestObj := new(Harvest)
+	tarFileName := "harvest.tar.gz"
+	utils.RemoveSafely(tarFileName)
+	err := utils.DownloadFile(tarFileName, r.path)
+	if err != nil {
+		panic(err)
+	}
+	log.Println("Downloaded: " + r.path)
+	log.Println("Check and remove harvest ")
+	if utils.FileExists(HarvestHome) && !harvestObj.AllStopped() {
+		harvestObj.Stop()
+	}
+	unInstallOutput := utils.Run("rm", "-rf", HarvestHome)
```

Comment:
scary 👻 
maybe double check that `HarvestHome` is valid or maybe it doesn't matter if the file system is accidentally removed